packages/api/endpoints/execution-status.js

```diff
@@ -1,9 +1,12 @@
 'use strict';
 
 const router = require('express-promise-router')();
-const aws = require('@cumulus/common/aws'); // important to import all to allow stubbing
+const {
```

Comment:
Why are there two Step Function related files? What's the difference?

Reply:
`@cumulus/aws-client/StepFunctions` is a direct migration of `@cumulus/common/StepFunctions`, whereas `@cumulus/aws-client/step-functions`is a decomposition of the step function related utilities that were in `@cumulus/common/aws`. The two files could probably be merged, but it seemed like there was some reason that you wrote `@cumulus/common/StepFunctions` to act like a "class" (or module) rather than a file that import specific helpers from. If you think they would be better combined, then that's fine with me

Reply:
This got addressed in the other PR: https://github.com/nasa/cumulus/pull/1424/commits/2d385297f11ed04b8009062d9c9b128bbc7a8dd4